### PR TITLE
feat: add tests for tutorial 1,2,3

### PR DESCRIPTION
**Pull request by kushthedude**
Signed-off-by: kushthedude <kushthedude@gmail.com>

### CODE REVIEW

**Comment by kushthedude** [comment]
@fgiroud Can you please review the structure of e2e tests, if it looks good then I will push ahead the further tutorials too!

**Comment by fgiroud** [comment]
Hi @kushthedude looks good so far, but there's not much to review at this point, feel free to push more commits and ask for another review

**Comment by kushthedude** [comment]
@fgiroud, do take a look at this! two more tutorials are on the way once this gets in 🙂

File: main/tests/cypress/cypress/integration/tutorial/importing-data-into-openrefine.spec.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// This spec is implementation for chapter 02 of following tutorial https://librarycarpentry.org/lc-open-refine/02-importing-data/index.html
+
+describe(__filename, function () {
+  it('Create your first OpenRefine project (using provided data)', function () {
```
Comment (fgiroud) [inline main/tests/cypress/cypress/integration/tutorial/importing-data-into-openrefine.spec.js:4]: This test doesn't contains any assertions, you might want to add a few checks to ensure the grid is as expected

File: main/tests/cypress/cypress/integration/tutorial/layout-of-openrefine.spec.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// This spec is implementation for chapter 03 of following tutorial https://librarycarpentry.org/lc-open-refine/02-importing-data/index.html
```
Comment (fgiroud) [inline main/tests/cypress/cypress/integration/tutorial/layout-of-openrefine.spec.js:1]: This is probably https://librarycarpentry.org/lc-open-refine/03-working-with-data/index.html

File: main/tests/cypress/cypress/integration/tutorial/layout-of-openrefine.spec.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    ]);
+
+    cy.get('#summary-bar').should('to.contain', '1001 records');
+
+    //       Click both the Rows and Records options and observe how the numbers of Rows and Records are equal
+
+    cy.get('span[bind="modeSelectors"]').contains('rows').click();
+
+    cy.get('#summary-bar').should('to.contain', '1001 rows');
+  });
```
Comment (fgiroud) [inline main/tests/cypress/cypress/integration/tutorial/layout-of-openrefine.spec.js:63]: Missing the part "Choosing a good separator" ?

Comment (meenal06) [reply main/tests/cypress/cypress/integration/tutorial/layout-of-openrefine.spec.js:63]: It is a theoretical section, which has nothing practical for cypress to asserting rather its for ease of user.

File: main/tests/cypress/cypress/integration/tutorial/layout-of-openrefine.spec.js (hunk 2 above)
Comment (meenal06) [reply main/tests/cypress/cypress/integration/tutorial/layout-of-openrefine.spec.js:1]: Ooops fixing

File: main/tests/cypress/cypress/integration/tutorial/importing-data-into-openrefine.spec.js (hunk 1 above)
Comment (meenal06) [reply main/tests/cypress/cypress/integration/tutorial/importing-data-into-openrefine.spec.js:4]: Sure👍

**Comment by kushthedude** [comment]
Sure doing all the requested changes right over!

File: main/tests/cypress/cypress/integration/tutorial/layout-of-openrefine.spec.js (hunk 3 above)
Comment (kushthedude) [reply main/tests/cypress/cypress/integration/tutorial/layout-of-openrefine.spec.js:63]: That part has nothing to perform, but its just a deduction on the readers' side

File: main/tests/cypress/cypress/integration/tutorial/importing-data-into-openrefine.spec.js (hunk 1 above)
Comment (kushthedude) [reply main/tests/cypress/cypress/integration/tutorial/importing-data-into-openrefine.spec.js:4]: done

**Comment by kushthedude** [comment]
I have done the requested changes @fgiroud , please do review !

**Comment by kushthedude** [comment]
Any update here @fgiroud

**Review by fgiroud — APPROVED** [review]

**Pull request merged**